components/StateHandlers/HydrationStateHandler.tsx

```diff
@@ -18,11 +19,29 @@ export default function HydrationStateHandler() {
       } else {
         getInstalledWallets(false);
       }
-      accounts.map((a) => getXmtpClient(a));
 
-      // accounts.map((address) => {
-      //   refreshProfileForAddress(address, address);
-      // });
+      // Fetching persisted conversation lists for all accounts
+      // We may want to fetch only the selected account's conversation list
+      // in the future, but this is simple for now, and want to get feedback to really confirm
```

Comment:
What are we confirming?

Reply:
Confirming actual timing of real users

Difficult to gauge in dev, but benchmarks on the SDK side are good